requirements/main.in

```diff
@@ -14,3 +14,5 @@ click
 kubernetes
 jsonschema
 requests
+docker-image-py
```

Comment:
what is this dependency and where do we use it?

Reply:
It is a reimplementation of the Go Docker Image Reference parser in Python, and we use it to do the parsing of the generic reference string into host, repo, name, tag, and digest.  I figured that if someone else's implementation already existed it was better than using my own.